src/lib/components/events/event-item.svelte

```diff
@@ -173,6 +173,7 @@
     margin: 0;
     padding: 0;
     padding: 12px 24px;
+    width: auto;
```

Comment:
Did this work?

Reply:
This didn't do much of anything, so I reverted it back in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/280/commits/8cf969c6b0ae3ad65577ebb9c168ddf62242fa3e>.